website/docs/tutorials/basic/your_first_app/5_defaults.md

```diff
@@ -68,6 +68,73 @@ $ python my_app.py ~db
 {}
 ```
 
+### Composition order of primary config
+Your primary config can contain both config values and a Defaults List.
+In such cases, you should add the `_self_` keyword to your defaults list to specify the composition order of the config file relative to the items in the defaults list.
+
+* If you want your primary config to override the values of configs from the Defaults List, append `_self_` to the end of the Defaults List.
+* If you want the configs from the Defaults List to override the values in your primary config, insert `_self_` as the first item in your Defaults List.
+ 
+
+<div className="row">
+
+<div className="col col--6">
+
+```yaml title="config.yaml" {3}
+defaults:
+  - db: mysql
+  - _self_
+
+db:
+  user: root
+```
+</div>
+<div className="col  col--6">
+
+```yaml title="Result config: db.user from config.yalm" {4}
+db:
+  driver: mysql  # db/mysql.yaml
+  pass: secret   # db/mysql.yaml 
+  user: root     # config.yaml
+
+
+```
+</div>
+<div className="col col--6">
+
+```yaml title="config.yaml" {2}
+defaults:
+  - _self_
+  - db: mysql
+
+db:
+  user: root
+```
+</div>
+<div className="col  col--6">
+
+```yaml title="Result config: All values from db/mysql" {4}
+db:
+  driver: mysql # db/mysql.yaml
+  pass: secret  # db/mysql.yaml
+  user: omry    # db/mysql.yaml
+
```

Comment:
I don't think so.
1. It will make this entire page much more complicated (we will need another config group).
2. I am showing in in the schema page where it makes sense.